.release-notes/split-filepath.md

```diff
@@ -0,0 +1,63 @@
+## Split `FilePath` construction into two methods
+
+`FilePath` previously had only one constructor, the default `create`, which used a union of `(AmbientAuth | FilePath)`. The first half of this union `AmbientAuth` could never `error`, while the second `FilePath` could `error`. By splitting construction into two methods, we now have a default `create` constructor which cannot `error` and a new `from` constructor which can `error`. The default `create` constructor now accepts a new "files" package root authority `FileAuth` as well as the global root authority `AmbientAuth`, while the new `from` constructor uses `FilePath`.
+
+The result of this change is that three user changes are needed, namely around `create`, `from`, and `mkdtemp`. Any place where previously `AmbientAuth` was accepted now also accepts `FileAuth`.
+
+Before `create` using `AmbientAuth` to now `create` using `AmbientAuth` or `FileAuth` -- note can no longer fail:
```

Comment:
These lines and the equivalents below read a bit weird to me, perhaps adding some punctuation would help?